docusaurus/website/siteConfig.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+const siteConfig = {
+  title: 'OpenShift Machine API',
+  tagline: 'Documentation of the OpenShift Machine API',
+  url: '', // Your website URL
+  baseUrl: '/', // Base URL for your project */
+  // For github.io type URLs, you would set the url and baseUrl like:
+  //   url: 'https://facebook.github.io',
+  //   baseUrl: '/test-site/',
+
+  // Used for publishing and more
+  projectName: 'machine-api-operator',
+  organizationName: 'openshift',
+  // For top-level user or org sites, the organization is still the same.
+  // e.g., for the https://JoelMarcey.github.io site, it would be set like...
+  //   organizationName: 'JoelMarcey'
+
+  // For no header links in the top nav bar -> headerLinks: [],
+  headerLinks: [
+    { doc: 'intro', label: 'Documentation' },
+  ],
+
+  /* path to images for header/footer */
+  /*
+  headerIcon: 'img/favicon.ico',
+  footerIcon: 'img/favicon.ico',
+  favicon: 'img/favicon.ico',
+  */
+
+  /* Colors for website */
+  colors: {
+    primaryColor: '#000000',
```

Comment:
That's a boring primary 😉

Reply:
my original thought was to use a palette derived from colors at brand.redhat.com, but i focused more on content that style ;)

Reply:
It is very minimalistic and I like it personally :)